core/src/kmx/kmx_plus.h

```diff
@@ -648,8 +648,8 @@ typedef KMX_DWORD KMXPLUS_USET;
 struct COMP_KMXPLUS_USET {
   static const KMX_DWORD IDENT = LDML_SECTIONID_USET;
   COMP_KMXPLUS_HEADER header;
-  KMX_DWORD usetCount;
-  KMX_DWORD rangeCount;
+  KMX_DWORD_unaligned usetCount;
+  KMX_DWORD_unaligned rangeCount;
```

Comment:
Does this need to be clarified in the kmx-plus-file-format spec?

Reply:
https://github.com/keymanapp/keyman/blob/master/docs/file-formats/kmx-plus-file-format.md does not discuss alignment

Reply:
> does not discuss alignment

We could note that it is packed and readers should be aware of alignment issues. I can add a note in this PR. (It wouldn't have helped us here because we already knew; this just slipped through the cracks. But it may help other consumers in the future.)